make_site.py

```diff
@@ -232,6 +243,32 @@ def from_top_level(cls, index: int, title: str, children) -> 'Overview':
 with (DATA/'theories_index.yaml').open('r', encoding='utf-8') as h_file:
     theories = yaml.safe_load(h_file)
 
+@dataclass
+class Project:
+    name: str
+    organization: str
+    description: str
+    maintainers: List[str]
+    stars: int 
+
+github = Github()
+
+with (DATA/'projects.yaml').open('r', encoding='utf-8') as h_file:
+    oprojects = yaml.safe_load(h_file)
+
+projects = []
+for project in [project for project in oprojects if 'display' not in oprojects[project] or oprojects[project]['display']]:
```

Comment:
Actually I wonder if the code wouldn't be clearer with something like
```python
for name, project in oprojects.items():
    if oprojects[project].get('display', True):
        ...
```
I would even build a `Project` constructor consuming `name` and `project`, but that's because I like OOP too much.

Reply:
Yes, I think (something like) that is clearer. Changed.